libr/core/anal.c

```diff
@@ -239,6 +239,17 @@ R_API void r_core_anal_autoname_all_fcns(RCore *core) {
 		}
 	}
 }
+
+static bool blacklisted_word(char* name) {
+	const char * list[] = {"__stack_chk_guard", "__stderrp", "__stdinp", "__stdoutp", 
+	"_DefaultRuneLocale"};
+	int i;
+	for (i = 0; i < sizeof(list)/sizeof(list[0]); i++) {
+        	if (strstr(name, list[i]) != NULL) return true;
```

Comment:
add braces, remove the unnecessary != NULL and add spaces before ( and between /, etc